scripts/irods/test/test_configuration_reload.py

```diff
@@ -97,18 +98,20 @@ def test_reload_command__issue_6529(self):
                         indent=4, separators=(',', ': ')))
 
             # Invoke irods-grid reload --all
-            _, out, _ = self.admin.assert_icommand(
-                "irods-grid reload --all", 'STDOUT_MULTILINE')
-
-            # Load the output of the command
-            obj = json.loads(out)
-            changes = obj["hosts"][0]["configuration_changes_made"]
-
-            # And check the properties such that they are what we expect
-            self.assertTrue(len(changes) == 1)
-            self.assertTrue(changes[0]["path"] == f"/{PROP_NAME}")
-            self.assertTrue(changes[0]["op"] == "add")
-            self.assertTrue(changes[0]["value"] == "Yes?")
+            with session.make_session_for_existing_admin() as admin_session:
+                admin_session.environment_file_contents = IrodsConfig().client_environment
```

Comment:
Still not sure that I understand the reason for this. Does this not work if we don't load the `IrodsConfig().client_environment` into the `admin_session.environment_file_contents`?

Reply:
Yeah, irods-grid doesn't work without this line